ArrayOfMultiples.rb

```diff
@@ -0,0 +1,32 @@
+=begin 
+Create a function that takes two numbers as arguments (num, length) and returns an array of multiples of num until the array length reaches length.
+
+Examples
+
+array_of_multiples(7, 5) ➞ [7, 14, 21, 28, 35]
+
+array_of_multiples(12, 10) ➞ [12, 24, 36, 48, 60, 72, 84, 96, 108, 120]
+
+array_of_multiples(17, 6) ➞ [17, 34, 51, 68, 85, 102]
+Notes
+Notice that num is also included in the returned array.
+
+Algorithm:
+1. Given a two numbers, First number is the multiplication number and the second number is the range limit from 1.. n. 
+2. Then the first number is multiplied with all the number is that range. 
+3. And the result is being stored and shown
+
+=end 
+
+def array_of_multiples(number, range)
+  multiplied = []
+  for i in 1..range 
+    multiplied.push(number * i)
+  end
+  multiplied
```

Comment:
this is good. you could have also done this in one line with something like this:

```
(1..range).map{|e| e * number}
```

Reply:
Yeah the map function, i had on my mind!

Reply:
cool. let's update this pr so we can merge this when this is updated

Reply:
I can't find this file in on this branch. what should i do? do i have to create a new file and update it ?

Reply:
i have created a new file and uploaded it

Reply:
you don't need to create a new file. you just need to checkout to this branch and you will see this file there in that branch. you will only see this file if you are in that branch. 

then in there you just have to edit this same file. No need to create a new file. Once you make the change you commit that update and you will be pushing that up. 

@Anishluke92